index.html

```diff
@@ -18,8 +19,7 @@
     <title>Travel Blog</title>
   </head>
 
-  <!-- body -->
-  <body>
+</head>
```

Comment:
There are a few tags scattered that have the wrong indentation or might not be needed (I think there are two head tags and a few body tags. Might be worth checking through for these.